atest/robot/core/filter_by_names.robot

```diff
@@ -1,6 +1,7 @@
 *** Settings ***
 Documentation      Tests for --test and --suite options.
 Resource           atest_resource.robot
+Test Tags    filter_by_names
```

Comment:
Please remove. We don't generally use tags like this. We do use tags for other purposes, though, and unnecessary tags like this are just a distraction. If tags like this were used, the value should be aligned to match other setting values.

Reply:
test tags removed. Just forgot to remove them as I'm using those to debug only those particular tests using "--include" switch